### DISCUSSION TITLE
Problems after moved to a Macbook M1

### DISCUSSION THREAD

**Discussion by elfeffe** [livewire/livewire#4927] [1. Help] [answered]
I have all the PHP code inside the Mac, and then I run a Parallels VM with Ubuntu.
It used to work for years.
Today I had several problems, including changing sendfile to off, because It was not serving CSS and JS files.

After that Livewire stopped working, it has error 419 and says that the page has expired.

Has anyone experienced this problem after switching to ARM processor?

I know it's out of scope here, but the sendfile issue was related to the new ARM processor in some way (it's about the way the system shares the files with the VM).

Can it be related to Ubuntu 20.04? I has 18.04 before and had to install it all again because of the ARM CPU.

**Comment by NBA707** [comment, marked as answer]
Did you check your access and error log (assuming Apache 2.4) in your VM?   /var/log/apache2

Laravel + Apache 2.4 works great on my M1 under Parallels.   No real difference from the Intel Mac I was using before.

Not sure where would be a better place to ask this question.  Seems like Apache is not configured correctly if it's not serving CSS/JS.

**Reply by elfeffe** [reply]
It was a very silly mistake :(
I knew it was not the best place to ask, but I broke my head with this M1 reinstall.
It was not related.
Thank you anyway.

**Reply by NBA707** [reply]
Glad you figured it out.
